mx.pwmc.Svgvi.json

```diff
@@ -0,0 +1,123 @@
+{
+    "app-id": "mx.pwmc.Svgvi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
+    "sdk": "org.gnome.Sdk",
+    "writable-sdk": true,
+    "command": "mx.pwmc.Svgvi",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options": {
```

Comment:
these are all defaults in this version and you should remove them.

Reply:
Ok. Done.